test/jdk/javax/swing/JComboBox/DisabledComboBoxFontTestAuto.java

```diff
@@ -99,11 +101,10 @@ private static void testMethod() throws IOException {
         ImageIO.write(disabledImage2, "png", new File(testDir
```

Comment:
We are saving these images even in the case where test passes. In my local macOS run i see 16 images getting saved in JTwork folder. This will end up taking good amount of space in our CI test systems.

We should save image only in case of test failure foe debugging purpose.

Also i see images with `Mac OS X***.png` filename and all of them are just blank and there is no combobox. Is that fine?

Reply:
> We are saving these images even in the case where test passes. In my local macOS run i see 16 images getting saved in JTwork folder. This will end up taking good amount of space in our CI test systems.
We should save image only in case of test failure foe debugging purpose.

Updated to save the image only when test fails.

> Also i see images with Mac OS X***.png filename and all of them are just blank and there is no combobox. Is that fine?

Even I also observed the same but not sure about this.